kyuubi-server/src/main/scala/org/apache/kyuubi/sql/parser/trino/KyuubiTrinoFeAstBuilder.scala

```diff
@@ -112,6 +112,6 @@ class KyuubiTrinoFeAstBuilder extends KyuubiTrinoFeBaseParserBaseVisitor[AnyRef]
   }
 
   override def visitTypesFilter(ctx: TypesFilterContext): List[String] = {
-    ctx.stirngInValue().asScala.map(v => unescapeSQLString(v.getText)).toList
+    ctx.STRING().asScala.map(v => unescapeSQLString(v.getText)).toList
```

Comment:
maybe `stirngInValue ` is better, `STRING` is more like a singular string than a list, But I'ok for now change